File: app/src/main/java/com/moudle/app/TestRoundActivity.java
package com.moudle.app;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.ImageView;

import com.moudle.app.view.RoundDrawable;

/**
 * 测试圆形图片
 */
public class TestRoundActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test_round);

        ImageView iv_head = findViewById(R.id.iv_head);

        Bitmap bitmap = BitmapFactory.decodeResource(getResources(), R.drawable.iv_scape);

        iv_head.setImageDrawable(new RoundDrawable(bitmap));
    }
}

File: app/src/main/java/com/moudle/app/activity/TestRoundActivity.kt
package com.moudle.app.activity

import android.graphics.BitmapFactory
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.moudle.app.R
import com.moudle.app.databinding.ActivityTestRoundBinding
import com.moudle.app.view.RoundDrawable

/**
 * 测试圆形图片
 */
class TestRoundActivity : AppCompatActivity() {
    private lateinit var roundBinding: ActivityTestRoundBinding

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        roundBinding = ActivityTestRoundBinding.inflate(layoutInflater)
        setContentView(roundBinding.root)

        val bitmap = BitmapFactory.decodeResource(resources, R.drawable.iv_scape)
        roundBinding.ivHead.setImageDrawable(RoundDrawable(bitmap))
    }
}